test/components/analytics/preview.spec.js

```diff
@@ -10,12 +10,18 @@ const analyticsPreview = {
     {
       users: { num_managers: { recent: 0, total: 0 } },
       forms: { num_forms: { recent: 0, total: 0 } },
-      submissions: { num_submissions_received: { recent: 0, total: 0 } }
+      submissions: {
+        num_submissions_received: { recent: 0, total: 0 },
+        num_submissions_from_web_users: { recent: 0, total: 0 }
+      }
     },
     {
       users: { num_managers: { recent: 1, total: 1 } },
       forms: { num_forms: { recent: 1, total: 1 } },
-      submissions: { num_submissions_received: { recent: 1, total: 1 } }
+      submissions: {
+        num_submissions_received: { recent: 1, total: 1 },
+        num_submissions_from_web_users: { recent: 0, total: 0 }
```

Comment:
I made all the numbers here 1 in order to differentiate this project from the other project. That way you can be sure that the numbers are coming from the second project, not the first.